lib/menu.rb

```diff
@@ -0,0 +1,38 @@
+require 'csv'
+# Can't figure out how to put csv files in a separate folder.
+# It always seems to return an error.
+
+class Menu
```

Comment:
In this project, something as simple as a Hash or array of Dish objects is fine to represent the menu.